bundles/org.openhab.ui/web/src/components/widgets/layout/oh-layout-page.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <template v-if="config.layoutType !== 'fixed'">
+    <template v-if="config.layoutType === 'responsive'">
```

Comment:
As stated above it should be:
```suggestion
    <template v-if="!config.layoutType || config.layoutType === 'responsive'">
```
since the layout types were introduced with 3.1 and pages created before don't have it and for backwards compatibility should still be treated as responsive types.
```